### ISSUE TITLE
add support for golang

### ISSUE THREAD

**Issue by luisdavim** [garabik/grc#122]
Something like this would colorise the output of `grc go test -v ./..`:

```
regexp==== RUN .*
colour=blue
-
regexp=--- PASS: .*
colour=green
-
regexp=^PASS$
colour=green
-
regexp=^(ok|\?) .*
colour=magenta
-
regexp=--- FAIL: .*
colour=red
-
regexp=[^\s]+\.go(:\d+)?
colour=cyan
```

**Comment by sir-farfan** [comment]
This configuration is working fine for me so far :)

**Comment by garabik** [comment]
fixed in 023c88b37b5a7da29d49b37fef26c6171549692a
